### PR TITLE
fix/update-draggable-component

### PR DESCRIPTION
**Pull request by Sammuel09**
- fix draggable list to update when the model value changes

## Summary by CodeRabbit

- **New Features**
	- Enhanced the draggable functionality with a new event handler to better detect and manage move events.

- **Refactor**
	- Improved the internal handling of draggable lists for efficiency and responsiveness.

### CODE REVIEW

**Review by kutsaniuk — COMMENTED** [review]
LGTM!

File: packages/vue/src/Elements/Draggable/OcDraggable.vue [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
-});
+    "update:modelValue": [],
+    "detectMove": null
+  },
+);
+
+
+const list = computed(() => {
+  return props.modelValue
+})
```
Comment (otaviolbarbosa) [inline packages/vue/src/Elements/Draggable/OcDraggable.vue:18]: Is there a reason to wrap props.modelValue in a computed property?

Comment (otaviolbarbosa) [reply packages/vue/src/Elements/Draggable/OcDraggable.vue:18]: Sounds good, @Sammuel09

**Review by otaviolbarbosa — APPROVED** [review]
LGTM

**Pull request merged**